src/main/java/frc/robot/vision/rawdata/RawAprilTagVisionData.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.vision.rawdata;
+
+import edu.wpi.first.apriltag.AprilTag;
+import edu.wpi.first.math.geometry.Pose3d;
+
+public class RawAprilTagVisionData extends RawVisionData {
+
+	private final double aprilTagHeight;
+	private final double distanceFromAprilTag;
+	private final AprilTag aprilTag;
+
+	public RawAprilTagVisionData(
+		Pose3d estimatedPose,
+		double timestamp,
+		boolean isDataValid,
+		double aprilTagHeight,
+		double distanceFromAprilTag,
+		AprilTag aprilTag
+	) {
+		super(estimatedPose, timestamp, isDataValid);
+		this.aprilTagHeight = aprilTagHeight;
+		this.distanceFromAprilTag = distanceFromAprilTag;
+		this.aprilTag = aprilTag;
+	}
+
+	public double getAprilTagHeight() {
+		return aprilTagHeight;
+	}
+
+	public double getDistanceFromAprilTag() {
+		return distanceFromAprilTag;
+	}
+
+	public AprilTag getTrackedAprilTag() {
+		return aprilTag;
+	}
```

Comment:
But It can be more than one no?

Reply:
We can't get that data out in all of our cameras (limelight, for examples, just returns the tag id as -1 when it combines more than one target)

Reply:
it really depends on many things, I guess there could be an ID for when there are multiple IDs, or we could just ignore a filter if the ID is -1.

Reply:
but that's for filters pr